src/main.py

```
@@ -333,59 +289,88 @@ async def _run_chat(env, message: str, history: list) -> dict:
+                reply = (
+                    response_output.get('response') or 
+                    response_output.get('reply') or 
+                    response_output.get('text') or 
+                    response_output.get('content')
+                )
+            
+        if reply is None:
+            print(f"AI Fallback extraction needed. Full response: {ai_response}")
+            reply = "I received a response but couldn't parse the text. Please check the logs."
+            
     except Exception as ai_error:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)